packages/ubuntu_desktop_installer/test/choose_your_look_page_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:gsettings/gsettings.dart';
+import 'package:provider/provider.dart';
+import 'package:ubuntu_desktop_installer/app_theme.dart';
+
+import 'package:ubuntu_desktop_installer/pages/choose_your_look_page.dart';
+import 'package:ubuntu_desktop_installer/widgets/option_card.dart';
+
+import 'simple_navigator_observer.dart';
+
+void main() {
+  final themeSettings = GSettings(schemaId: 'org.gnome.desktop.interface');
```

Comment:
we should not touch the system theme in settings, but we could start with this and setup mockito later. it's not 100% compatible with dart:ffi so it needs some little manual tweaks.